DESCRIPTION

```diff
@@ -43,7 +43,8 @@ Suggests:
     mlr3measures,
     praznik,
     rpart,
-    testthat
+    testthat,
+    caret
```

Comment:
Is this already implemented in {tidymodels}? I am worried about adding this to suggest because {caret}'s lifecycle is coming to its end in the foreseeable future.

Reply:
caret is only used for comparison in tests and not actually loaded by the filter, but I can have a look if they copied the behaviour to somewhere in another package.